serpentTools/objects/xsdata.py

```diff
@@ -115,8 +111,80 @@ def hasExpectedData(self):
 
         return True
 
+    def tabulate(self, mts='all', colnames=None):
+        """ Returns a pandas table, for pretty tabulation in Jupyter
+        notebooks.
+
+        Parameters
+        ----------
+        mts: int, string, or list of ints
+            If it's a string, it should be 'all', which is default.
+            A single int indicates one MT reaction number.
+            A list should be a list of MT numbers to plot.
+        colnames: any type with string representation
+            Column names for each MT number, if you'd like to change them.
+
+        Returns
+        -------
+        Pandas Dataframe version of the XS data.
+
+        Raises
+        ------
+        ImportError:
+            if you don't have pandas.
+        TypeError:
+            if MT numbers that don't make sense come up
+        """
+        try:
+            import pandas as pd
+        except ModuleNotFoundError:
+            print("This requires the pandas package.")
+            return None
```

Comment:
Would you rather just let the import error be raised? It's in the docstring, so one thing needs to be updated for consistency

Reply:
Fair enough.